VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -8,6 +8,9 @@ namespace VulnerableApplication.Controllers
 {
     public class HomeController : Controller
     {
+        public static string SonarPassword = "foo";
+        public static string GitHubPassword = "bar";
```

Comment:
[CODE_SMELL] 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.